### PR TITLE
Refactor Backend.setupSchema to return schema info

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

File: versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Backend.java [hunk 1]
```diff
@@ -20,12 +20,17 @@
...
 
-  void setupSchema();
+  /**
+   * Set up the schema for the backend and create the necessary tables / collections / column
+   * families.
+   *
+   * @return any useful information about the schema setup, such as the database name; to be
+   *     displayed when the application starts.
+   */
+  String setupSchema();
```
Comment (dimas-b) [inline versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Backend.java:30]: nit: `@Nonnull`?

**Review by dimas-b — APPROVED** [review]

File: versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Backend.java (hunk 1 above)
Comment (adutra) [reply versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Backend.java:30]: Or maybe allow nulls to be returned?

Comment (adutra) [reply versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Backend.java]: I went with `Optional` in the end.

**Review by dimas-b — APPROVED** [review]

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/PersistProvider.java [hunk 2]
```diff
@@ -167,7 +159,11 @@ public Persist producePersist(MeterRegistry meterRegistry) {
       cacheInfo = "without objects cache";
     }
 
-    LOGGER.info("Using {} version store{}, {}", versionStoreType, info, cacheInfo);
+    LOGGER.info(
+        "Using {} version store{}, {}",
+        versionStoreType,
+        info.map(s -> " (" + s + ")").orElse(""),
```
[the remark this answers is not in the document]
Comment (adutra) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/PersistProvider.java:165]: Not sure, the comma should be printed regardless, no?

[the remark this answers is not in the document]
Comment (dimas-b) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/PersistProvider.java:165]: ah, yes, my mistake

**Pull request merged**